CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+First of all, thank you for helping us, improve the Thanos!
+
+When contributing not obvious change to Thanos repository, please first
```

Comment:
Please first discuss any non obvious changes via issue, slack or other method with the owners of this repository before making a change.